### ISSUE TITLE
about visual effect of the model

### ISSUE THREAD

**Issue by zhui064** [YBIGTA/pytorch-hair-segmentation#36]
I have run the script evaluate.py on Figaro dataset,  using the model in 
  ./model/pspnet_resnet101_sgd_lr_0.002_epoch_100_test_iou_0.918.pth. 
I found that the visual effect of Figaro images is not as good as the effect showed in paper to a certain extent, especially in the boundary and hair region.
Did you train the model without mask-image gradient consistency loss in Figaro dataset , and compare the result with the model train with the loss.

**Comment by Jooong** [comment]
@zhui064 If you're talking about [this paper](https://arxiv.org/pdf/1712.07168.pdf), no.  It's simple implementation of pspnet using resnet101 as backbone. We tried to reproduce that paper using gradient consistency loss, but failed so we didn't push codes related to gradient consistency loss.
